### DISCUSSION TITLE
Please update the compatibility matrix

### DISCUSSION THREAD

**Discussion by bwilliams-halix** [abacritt/angularx-social-login#706] [General]
I was trying to update the lib to be compatible with angular 14. The matrix says "@abacritt/angularx-social-login:1 | 13, 14, 15". However, 1.3+ appears to be Angular 15+. 1.2.6 appears to be the last compatible with Angular 14. Just a suggestion to break out that line of the matrix more and provide ranges of 1.x versions compatible with each 13, 14, and 15. 

Can close the discussion immediately upon reading. Thanks.

**Comment by Heatmanofurioso** [comment]
@bwilliams-halix Thanks for informing on this.
Gonna update it on my next PR, and will close this by then
